### DISCUSSION TITLE
which name should give to audio wav files inside wavs folder?

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1738] [General Q&A]
???.wav
???.wav
???.wav
???.wav

**Comment by p0p4k** [comment]
I would suggest you to read the documentation first. Read as much as you can first, search the discussions for similar questions in the past and then ask if you are still stuck. Thanks a lot. 
 https://tts.readthedocs.io/en/latest/formatting_your_dataset.html
